src/main/kotlin/sliv/tool/menubar/view/MenuBarView.kt

```diff
@@ -6,11 +6,12 @@ import tornadofx.*
 
 class MenuBarView : View() {
     private val mainController: MainController by inject()
+    val importer = find<ImporterView>()
 
     override val root = hbox {
         button("Import project") {
             action {
-                ImporterView().openModal()
+                importer.openModal()
                 mainController.importTestData()
```

Comment:
It is no more needed to load test data. Remove this and `importTestData` function

Reply:
Remove `importTestData` function too